src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/_item.html.twig

```diff
@@ -17,6 +17,8 @@
         {{ item.vars.value.unitPrice|sylius_price }}
     </td>
     <td class="col-md-1" style="text-align: right">
-        {{ item.vars.value.total|sylius_price }}
+        {% set regularPrice = item.vars.value.unitPrice * item.vars.value.quantity %}
```

Comment:
We shouldn't perform calculations here. Temporary we can extract it to a Twig helper and then we can think about more clean solution. What is more we should display value `unitsTotal + taxes` but the above will display value without tax.